orchestrator/careplanservice/config.go

```diff
@@ -12,6 +12,8 @@ func DefaultConfig() Config {
 type Config struct {
 	FHIR    coolfhir.ClientConfig `koanf:"fhir"`
 	Enabled bool                  `koanf:"enabled"`
+	// Defaults to false, should not be set true in Test or Prod
+	AllowUnmanagedFHIROperations bool `koanf:"allow_unmanaged_fhir_operations"`
```

Comment:
underscores break koanf, since it'll think it's a sub-struct (meaning nested Allow, Unmanaged, Fhir, and Operations structs), since it uses that delimiter for nesting when specifying them as environment variables.

Reply:
Good to know, thanks. Will update and fix the e2e test